src/Http/Controllers/CP/Collections/CollectionTreeController.php

```diff
@@ -38,6 +39,7 @@ public function update(Request $request, $collection)
 
         // Clone the tree and add the submitted contents into it so we can
         // validate URI uniqueness without affecting the real object in memory.
+        Tree::clearAllCachedFlattenedPages();
```

Comment:
Removing this line doesn't break any tests. We should add something to cover it.

This also feels like something that would need to happen for navs too, no?

Reply:
Good spot! I'll add a test

As for whether this is needed on navs too, I'll dig